### PR TITLE
Add git-nadiff convenience command

### PR DESCRIPTION
**Pull request by jforberg**
This lets you say things like 'git nadiff HEAD'.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -21,7 +21,7 @@ nadiff: $(obj)
 
 install: nadiff
 > mkdir -p ${PREFIX}/bin
-> cp -f nadiff ${PREFIX}/bin
+> cp -f nadiff git-nadiff ${PREFIX}/bin
 > chmod 755 ${PREFIX}/bin/nadiff
 
 uninstall:
```
Comment (adlertz) [inline Makefile:27]: Looks good but add git-nadiff to uninstall as well

**Review by adlertz — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (jforberg) [reply Makefile:27]: ok

File: Makefile [hunk 2]
```diff
@@ -21,11 +21,11 @@ nadiff: $(obj)
 
 install: nadiff
 > mkdir -p ${PREFIX}/bin
-> cp -f nadiff ${PREFIX}/bin
+> cp -f nadiff git-nadiff ${PREFIX}/bin
 > chmod 755 ${PREFIX}/bin/nadiff
```
Comment (adlertz) [inline Makefile:25]: Sorry, I forgot to add that we also need this for git-nadiff 
chmod 755 ${PREFIX}/bin/git-nadiff

**Comment by adlertz** [comment]
I'm closing this. I fetched your commit and did the chmod change, and fixed a typo in the commit message. It's now rebased onto master

**Pull request closed without merging**